x/supply/keeper/querier.go

```diff
@@ -1,56 +0,0 @@
-package keeper
-
-import (
-	"github.com/cosmos/cosmos-sdk/codec"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/crypto-com/chain-main/x/supply/types"
-	abci "github.com/tendermint/tendermint/abci/types"
-)
-
-// NewQuerier returns a new sdk.Keeper instance.
-func NewQuerier(k Keeper, legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
-	return func(ctx sdk.Context, path []string, req abci.RequestQuery) ([]byte, error) {
-		switch path[0] {
-		case types.QueryTotalSupply:
```

Comment:
Are these queriers not used?

Reply:
I don't think so. @calvinaco can clarify.

Reply:
checked with @calvinaco -- this legacy one wasn't being used or planned to be used by third parties